en/micro-integrator/docs/references/connectors/exporting-artifacts.md

```diff
@@ -0,0 +1,27 @@
+# Exporting artifacts
+In this document we focus on creating the Connector Exporter Project and Exporting the CApp. 
+
+## Creating Connector Exporter Project
```

Comment:
Same comment as above. I think this will be ###.

Reply:
Done